.ci/jenkins_job_build.sh

```diff
@@ -180,6 +180,11 @@ else
 	echo "WARN: Kata runtime is not installed"
 fi
 
+if [ -n "${VFIO_CI}" ]; then
+	"${ci_dir_name}/run.sh"
+	exit $?
```

Comment:
The `exit` won't get called strictly due to the `set -e`, so you could just simplify to:

```sh
[ -n "${VFIO_CI}" ] && "${ci_dir_name}/run.sh"
```